src/client/visualizations/bar-chart/improved-bar-chart/utils/splits.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2017-2018 Allegro.pl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Essence } from "../../../../../common/models/essence/essence";
+
+export function firstSplitRef(essence: Essence): string {
```

Comment:
should be define this as Essence method?

Reply:
This is meaningful operation in barchart scope. I don't like fat classes with every accessor method possible.

Reply:
ok